### PR TITLE
Adds example of session personas to UserInfo view in BrandGame

### PR DESCRIPTION
**Pull request by atreat**
This PR also fixes a bug when removing metadata in the MetadataHandler. 

The `lifespanId` was not being passed in correctly so only `permanent` records could be removed.

### CODE REVIEW

File: Sources/EmbraceCore/Public/Metadata/MetadataHandler.swift [hunk 1]
```diff
@@ -66,28 +66,14 @@ public class MetadataHandler: NSObject {
...
-        }
+        let lifespanContext = try currentContext(for: lifespan.recordLifespan)
 
         let record = try storage.addMetadata(
             key: key,
             value: validateValue(value),
             type: type,
             lifespan: lifespan.recordLifespan,
-            lifespanId: id
+            lifespanId: lifespanContext
```
Comment (NachoEmbrace) [inline Sources/EmbraceCore/Public/Metadata/MetadataHandler.swift:76]:
```suggestion
            lifespanId: lifespanId
```

File: Sources/EmbraceCore/Public/Metadata/MetadataHandler.swift [hunk 2]
```diff
@@ -185,6 +179,22 @@ extension MetadataHandler {
     }
 }
 
+extension MetadataHandler {
+    private func currentContext(for lifespan: MetadataRecordLifespan) throws -> String {
```
Comment (NachoEmbrace) [inline Sources/EmbraceCore/Public/Metadata/MetadataHandler.swift:183]:
```suggestion
    private func lifespanId(for lifespan: MetadataRecordLifespan) throws -> String {
```

File: Sources/EmbraceCore/Public/Metadata/MetadataHandler.swift [hunk 3]
```diff
@@ -144,11 +130,19 @@ public class MetadataHandler: NSObject {
...
+    ///  - lifespan: The lifespan of the metadata to remove.
+    ///
+    ///  - Throws: `MetadataError.invalidSession` if a metadata with a `.session` lifespan is removed when there's no active session.
     func remove(key: String, type: MetadataRecordType, lifespan: MetadataLifespan = .session) throws {
         try storage?.removeMetadata(
             key: key,
             type: type,
-            lifespan: lifespan.recordLifespan
+            lifespan: lifespan.recordLifespan,
+            lifespanId: try currentContext(for: lifespan.recordLifespan)
```
Comment (NachoEmbrace) [inline Sources/EmbraceCore/Public/Metadata/MetadataHandler.swift:145]:
```suggestion
            lifespanId: try lifespanId(for: lifespan.recordLifespan)
```

**Comment by NachoEmbrace** [comment]
Seems that `lifespanContext` is just the id.
Left some suggestions so we just use `lifespanId` so it's less confusing.

**Review by ArielDemarco — APPROVED** [review]
Given that most of the comments are related to naming, i'll proceed to merge this so we can unblock 6.2 release.
However, it'd be good to address the PR comments in another PR

**Pull request merged**